app/views/rankings/_ranking.html.erb

```diff
@@ -1,15 +1,18 @@
-<% unless leaderboard.empty? %>
-  <ol start="<%= (25*(params[:page].to_i-1))+1 if params[:page] %>">
-  <% leaderboard.each do |item| %>
-    <% if item.user %>
-      <li class="block <%= content_for_user(item.user) { "you" } %>">
-        <%= render :partial => 'users/user', :locals => {:user => item.user} %>
-        <div class="count"><%= pluralize(item.value.to_i, 'tomato') %></div>
-      </li>
-    <% end %>
+<div class="leaderboard_list">
+  <% unless leaderboard.empty? %>
+    <ol start="<%= (25*(params[:page].to_i-1))+1 if params[:page] %>">
+      <% leaderboard.each do |item| %>
+        <% if item.user %>
+          <li class="block <%= content_for_user(item.user) { "you" } %>">
```

Comment:
Actually you can't remove completely the helper function because it's used elsewhere.